Source/Rows/PickerRow.swift

```diff
@@ -75,7 +75,7 @@ open class PickerCell<T> : Cell<T>, CellType, UIPickerViewDataSource, UIPickerVi
     open func numberOfComponents(in pickerView: UIPickerView) -> Int {
         return 1
     }
-
+    
```

Comment:
Please remove this tab

Reply:
That should be a newline, or am I missing something? 😅